docs/docs/destinations/README.md

```diff
@@ -1,11 +1,11 @@
 # Overview
```

Comment:
Updated to fix old screenshots references, and clarify how to use destinations with component actions.